src/marshmallow_sqlalchemy/convert.py

```diff
@@ -45,15 +45,20 @@ class ModelConverter:
         postgresql.UUID: fields.UUID,
         postgresql.MACADDR: fields.String,
         postgresql.INET: fields.String,
+        postgresql.CIDR: fields.String,
         postgresql.JSON: fields.Raw,
         postgresql.JSONB: fields.Raw,
         postgresql.HSTORE: fields.Raw,
         postgresql.ARRAY: _postgres_array_factory,
         postgresql.MONEY: fields.Decimal,
+        postgresql.DATE: fields.Date,
+        postgresql.TIME: fields.Time,
         mysql.BIT: fields.Integer,
         mysql.YEAR: fields.Integer,
         mysql.SET: fields.List,
         mysql.ENUM: fields.Field,
+        mysql.INTEGER: fields.Integer,
```

Comment:
Does the fallback to `python_type` [here](https://github.com/marshmallow-code/marshmallow-sqlalchemy/blob/08c422349da60462ef88a97d413a05a484e1cd6f/src/marshmallow_sqlalchemy/convert.py#L170) get hit for these?